### DISCUSSION TITLE
Doubt publishing new JEDI Visual Component Library in Embarcadero's GetIt delphi 12

### DISCUSSION THREAD

**Discussion by raulbou** [project-jedi/jvcl#170] [General]
Hello, I want to make a big version change. We want to go from Delphi XE4 to Delphi 12. Our projects contain many developments with the Jedi libraries.

Do you plan to release compatibility with Delphi 12? Will this library be available with the Delphi IDE GetIt tool? Is there a date planned?

Thank you.

**Comment by obones** [comment]
Yes, it will be compatible with Delphi 12 but I have no date to communicate because I'm not Embarcadero.
Also, I don't have control over the GetIt package, it's Embarcadero managing it as well.

**Reply by raulbou** [reply]
Thank you very much for answering. I thought you had control over GetIt and not Embarcadero.
I understand that although it does not appear in getIt, we can install the libraries with the own installation method offered by the project.

Finally, regarding compatibility, is the current version of the Jedi libraries compatible with delphi 11.3?
The same thing happens here, GetIt offers a version compatible with 11.xx but if you install the delphi 11.3 demo, installation via getit is not available.


Greetings and sorry if the questions are not appropriate in this forum.

Thank you.
